### ISSUE TITLE
a small bug in "experiments/movingmnist/mnist_rnn_main.py"

### ISSUE THREAD

**Issue by igloooo** [sxjscience/HKO-7#8]
In the function "train" line 120, when creating a moving mnist factory,  the parameter ctx_num is omitted, while the parameter batch_size is still divided by the number of gpus. This will cause dimension mismatch when running the file with multiple gpus.

**Comment by sxjscience** [comment]
Yeah, thanks for pointing out this bug. I haven't tried using multiple GPUs for the MovingMNIST experiments.
